### PR TITLE
Topic 0 subject 3

### PR DESCRIPTION
**Pull request by M0narc**
Here's the Proxy!

### CODE REVIEW

File: Topic0Proxy/pom.xml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    <groupId>Topic0Proxy</groupId>
+    <artifactId>Topic0Proxy</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>6</source>
```
Comment (LeoBur) [inline Topic0Proxy/pom.xml]: Use java 8 or higher

File: Topic0Proxy/src/main/java/GlobantProxy/Main.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package GlobantProxy;
```
Comment (LeoBur) [inline Topic0Proxy/src/main/java/GlobantProxy/Main.java]: packages should be in lower case

File: Topic0Proxy/src/main/java/GlobantProxy/ProxyDB.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    private String user;
+    private String pass;
+
+    public ProxyDB(String nameDB, String user, String pass) {
+        this.realDB = realDB;
+        this.nameDB = nameDB;
+        this.user = user;
+        this.pass = pass;
+    }
+
```
Comment (LeoBur) [inline Topic0Proxy/src/main/java/GlobantProxy/ProxyDB.java]: Remove unnecesary empty lines

File: Topic0Proxy/pom.xml (hunk 1 above)
Comment (M0narc) [reply Topic0Proxy/pom.xml]: It has been modified, the IDE was supposed to be using java 8 but that was clearly a lie.

File: Topic0Proxy/src/main/java/GlobantProxy/Main.java (hunk 2 above)
Comment (M0narc) [reply Topic0Proxy/src/main/java/GlobantProxy/Main.java]: modified to globantproxy*.

File: Topic0Proxy/src/main/java/GlobantProxy/ProxyDB.java (hunk 3 above)
Comment (M0narc) [reply Topic0Proxy/src/main/java/GlobantProxy/ProxyDB.java]: Unnecesary lines removed!

File: Topic0Proxy/src/main/java/globantProxy/Main.java [hunk 4]
```diff
@@ -0,0 +1,11 @@
+package globantProxy;
+
+public class Main {
+    public static void main (String[] args){
+        ProxyInt db = new ProxyDB("MySql","franco","Qwerty123");
+            db.display();
```
Comment (LeoBur) [inline Topic0Proxy/src/main/java/globantProxy/Main.java]: Check indentation

**Pull request closed without merging**